### PR TITLE
[ALLUXIO-1860] Add the Portuguese version of documentation page "index.md" 

### PR DESCRIPTION
**Pull request by caadecarvalho**
Add the Portuguese version of documentation page "index.md" and other nested pages

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/pt/Running-Alluxio-on-EC2.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+Para `AWS EC2`, o `underfs` padrão é o `S3`. Você precisa logar em sua [console Amazon S3](http://aws.amazon.com/s3/), 
+criar um `S3 bucket` e escrever o nome deste `bucket` no campo `S3:Bucket` em `conf/ufs.yml`. Para utilizar 
+outro `under storage system`, configure o campo `Type` e as configurações correspondentes em `conf/ufs.yml`.
+
+Agora você pode iniciar o Alluxio `cluster` com o seu `under filesystem` escolhido dentro 
+`availability zone` através da execução do `script` em `deploy/vagrant`:
+
+{% include Running-Alluxio-on-EC2/launch-cluster.md %}
+
+Cado nó do `cluster` executa um Alluxio `worker` e o `AlluxioMaster` roda o Alluxio `master.
```
Comment (aaudiber) [inline docs/pt/Running-Alluxio-on-EC2.md]: missing ` after master

**Comment by aaudiber** [comment]
Thanks for this great translation work @caadecarvalho! I've left one minor comment, otherwise LGTM

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

**Comment by caadecarvalho** [comment]
Thanks, @aaudiber !

**Comment by aaudiber** [comment]
LGTM, thanks for the quick iteration

**Pull request merged**